x-pack/plugins/serverless_security/public/components/side_navigation/side_navigation.tsx

```diff
@@ -39,6 +39,7 @@ export const SecuritySideNavigation: React.FC = () => {
         />
       }
       closeFlyoutButtonPosition={'inside'}
+      titleDataTestSubj="svlSecNavTitle"
```

Comment:
can we use:
```suggestion
      headingProps={{
        'data-test-subj': 'securitySolutionNavHeading',
      }}
```
so we don't have to create an specific prop for it?

Reply:
Good point, updated in 1a45f95.